### PR TITLE
Support selected item HTML replacement

### PR DESCRIPTION
**Pull request by muan**
We need avatar + text for button value replacement in many places, most notably in account select menus.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -172,15 +172,15 @@ function updateLabel(item: Element, details: Element) {
   const button = details.querySelector('[data-menu-button]')
   if (!button) return
 
-  const text = label(item) || label(item.querySelector('[data-menu-button-text]'))
-  if (text) button.textContent = text
+  const content = label(item) || label(item.querySelector('[data-menu-button-text]'))
+  if (content) button.innerHTML = content
```
Comment (josh) [inline index.js]: I'd prefer this attribute be renamed or we support both text and html like `data-menu-button-text` and `data-menu-button-html`.

There's a bunch of escaping security gotchas if people think the attribute is text and will be escaped by the browser, but it turns out we're using innerHTML to set the value.

I think `data-menu-button-html` is _okay_, but it annoys me a little to have html embedded in attributes.

Comment (josh) [reply index.js]: Okay, maybe disregard this comment. I thought the value of the `data-menu-button-text` attribute was being used, not the inner contents.

Comment (josh) [reply index.js]: Dang, it is lol. Okay this is confusing hehe

File: index.js [hunk 2]
```diff
@@ -172,15 +172,29 @@ function updateLabel(item: Element, details: Element) {
...
-function label(el: ?Element): ?string {
+function labelHTML(el: ?Element): ?string {
   if (!el) return null
-  const text = el.getAttribute('data-menu-button-text')
-  if (text == null) return null
-  return text === '' ? el.textContent : text
+  const contentsEl = el.hasAttribute('data-menu-button-contents') ? el : el.querySelector('[data-menu-button-contents]')
+
+  if (!contentsEl) return null
+  return contentsEl.innerHTML
```
Comment (dgraham) [inline index.js]: `return contentsEl ? contentsEl.innerHTML : null`

**Pull request merged**